spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/HealthEndpoint.java

```diff
@@ -43,6 +45,16 @@
 	private long timeToLive = 1000;
 
 	/**
+	 * Flag to enable parallel invocation of health indicators.
+	 */
+	private boolean runInParallel = false;
+
+	/**
+	 * Number of threads to use for parallel invocation of health indicators.
```

Comment:
May want to explicitly document here that this should be greater than one if set.